.github/workflows/build.yml

```diff
@@ -1,10 +1,18 @@
 name: Build
 
 on:
+  pull_request:
```

Comment:
we're already building the code for other workflows
maybe we can somehow unify it and build only once so we don't spend time & resources?

Reply:
We need to do it, but it's for another PR.

Reply:
Maybe open a task for that?